### PR TITLE
test(demo-playwright): tests for `LineChart`

### PR DESCRIPTION
**Pull request by VidhiRambhia**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [X] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Addresses [# [4912 ](https://github.com/taiga-family/taiga-ui/issues/4912)]

## What is the new behavior?

This PR adds an e2e test (with playwright) for line chart

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+test.describe(`LineChart`, () => {
+    test(`should not show hint`, async ({page}) => {
+        await tuiGoto(page, `charts/line-chart`);
+        const example = new TuiDocumentationPagePO(page).getExample(`#line`);
+        const lineChart = example
+            .locator(`tui-line-chart-example-1 tui-line-chart .t-column`)
+            .first();
+
+        await lineChart.hover();
+        await expect(lineChart).toHaveScreenshot(`01-line-chart.png`);
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts:13]: It generates the following screenshot:


Please, change screenshot area
```suggestion
        await expect(example).toHaveScreenshot(`01-line-chart.png`);
```

Comment (nsbarsukov) [reply projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts:13]:
```ts
await tuiGoto(page, `charts/line-chart`);
const example = new TuiDocumentationPagePO(page).getExample(`#line`);

await example
    .locator(`tui-line-chart-example-1 tui-line-chart .t-column`)
    .first()
    .hover();

await expect(example).toHaveScreenshot(`01-line-chart.png`);
```

File: projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`LineChart`, () => {
+    test(`should not show hint`, async ({page}) => {
```
[the remark this answers is not in the document]
Comment (VidhiRambhia) [reply projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts:5]: Thanks, deleted it

File: projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        await tuiGoto(page, `charts/line-chart`);
+        const example = new TuiDocumentationPagePO(page).getExample(`#line`);
+        const lineChart = example
```
Comment (nsbarsukov) [inline projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts:8]: Actually, this variable keeps locator not for the whole chart but the part of it.
Should we rename variable ? 🤔 

```suggestion
        const chartColumn = example
```

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts (hunk 3 above)
Comment (VidhiRambhia) [reply projects/demo-playwright/tests/addon-charts/line-chart/line-chart.spec.ts:8]: Makes sense. Sorry for this!

**Pull request merged**